hassrelease/commands.py

```diff
@@ -150,3 +151,15 @@ def unmerged_docs(branch, release):
         print(pr.title)
         print(docs_pr.html_url)
         print()
+
+
+@cli.command(help='Generate credits page'
+                  ' (../home-assistant.io/source/developers/credits.markdown)')
+@click.option('-r', '--simul-requests', default=63, type=click.IntRange(min=1),
```

Comment:
During my little test 63 seems to be too high.

With

```bash
$ hassrelease credits -r 10
```

it will take a little longer till I hit the rate limit.